misc/diagrams.md

```diff
@@ -0,0 +1,29 @@
+# Diagramming Guidelines
+
+We use [Mermaid](https://mermaid.js.org/intro/) as our primary diagramming tool. Mermaid is “a JavaScript based diagramming and charting tool that renders Markdown-inspired text definitions to create and modify diagrams dynamically”. Mermaid enables us to:
```

Comment:
Non-blocking. It might be helpful to link the quote back to wherever it came from to give it more context.